packages/js-drive/lib/dpp/CachedStateRepositoryDecorator.js

```diff
@@ -281,6 +281,30 @@ class CachedStateRepositoryDecorator {
   async fetchSMLStore() {
     return this.stateRepository.fetchSMLStore();
   }
+
+  /**
+   * Fetch latest withdrawal transaction index
+   *
+   * @returns {Promise<number>}
+   */
+  async fetchLatestWithdrawalTransactionIndex() {
+    return this.stateRepository.fetchLatestWithdrawalTransactionIndex();
```

Comment:
We need to pass state transition execution context here and handle dry run I guess. Let's add a todo for now?